inputs/cli_test.go

```diff
@@ -53,3 +53,24 @@ func TestParseCLIArgsInputs(t *testing.T) {
 	assert.True(p.Answered)
 	assert.Equal("y", p.Answer)
 }
+
+func TestParseSelectCLIArgsInputs(t *testing.T) {
+	assert := assert.New(t)
+	p := newSimpleTextPrompter(InputSpec{
+		Type:    "select",
+		ID:      "s",
+		Options: []string{"simple", "advanced"},
+	})
+
+	err := ParseCLIArgsInputs(&mockInputsCollector{prompters: []Prompter{p}}, []string{})
+	require.NoError(t, err)
+	assert.False(p.Answered)
+
+	err = ParseCLIArgsInputs(&mockInputsCollector{prompters: []Prompter{p}}, []string{"--s", "simple"})
+	require.NoError(t, err)
```

Comment:
These two likes seem redundant given the four lines below, they test the same thing

Reply:
That supposed to be an error test. Probably got distracted after Copy-Paste.